src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskListTabPanel.tsx

```diff
@@ -3,28 +3,227 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Box, Link, Typography } from '@mui/material';
+import { Box, Link, useTheme } from '@mui/material';
+import { DataGrid, GridActionsCellItem, GridColumns, GridRowId, GridRowParams } from '@mui/x-data-grid';
+import { RoleEnum, Task, TaskPriority, TaskStatus, UserPreview } from 'shared';
+import { datePipe, fullNamePipe } from '../../../utils/pipes';
+import { GridColDefStyle } from '../../../utils/tables';
+import PauseIcon from '@mui/icons-material/Pause';
+import DeleteIcon from '@mui/icons-material/Delete';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import CheckIcon from '@mui/icons-material/Check';
+import { useAuth } from '../../../hooks/auth.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import React from 'react';
+
+//this is needed to fix some weird bug with getActions()
+declare global {
+  namespace React {
+    interface DOMAttributes<T> {
+      onResize?: ReactEventHandler<T> | undefined;
+      onResizeCapture?: ReactEventHandler<T> | undefined;
+      nonce?: string | undefined;
+    }
+  }
+}
 
 interface TaskListTabPanelProps {
   index: number;
   value: number;
+  tasks: Task[];
+  status: TaskStatus;
 }
 
 const TaskListTabPanel = (props: TaskListTabPanelProps) => {
-  const { value, index } = props;
+  const { value, index, tasks, status } = props;
+
+  const auth = useAuth();
+
+  if (!auth.user) return <LoadingIndicator />;
+
+  const disabled = auth.user.role === RoleEnum.GUEST;
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const theme = useTheme();
+
+  const renderNotes = () => <Link>See Notes</Link>;
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const moveToBacklog = React.useCallback(
+    (id: GridRowId) => () => {
+      console.log('move to backlog');
```

Comment:
Looks like we want these buttons to work in this ticket? If not do we have a ticket for that?